docs/aca/00-workshop-intro/3-dapr-integration.md

```diff
@@ -31,7 +37,7 @@ This pattern is named Sidecar because it resembles a sidecar attached to a motor
 
 We are going to enable Dapr for all Azure Container Apps in the solution, the Dapr APIs/Building Blocks used in this workshop are:
 
-* **Service to Service invocation**: "ACA Web App-Frontend" microservice invokes the "ACA WebAPI-Backend" microservice using Dapr sidecar.
-* **State Management**: "ACA WebAPI-Backend" stores data on Azure Cosmos DB and store email logs on Azure Table Storage using Dapr State Management building blocks.
+* **Service to Service invocation**: "ACA Web App-Frontend" microservice invokes the "ACA WebAPI-Backend" microservice using Dapr Service-to-service invocation building blocks.
+* **State Management**: "ACA WebAPI-Backend" stores data on Azure Cosmos DB and stores email logs on Azure Table Storage using Dapr State Management building blocks.
 *** Pub/Sub**: "ACA WebAPI-Backend" publishes messages to Azure Service Bus when a task is saved and the "ACA Processor-Backend" microservices consumes those messages and sends emails using SendGrid.
```

Comment:
space is needed between bullet line and **

Reply:
done